src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator {
+    private String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public void draw() {
+    }
```

Comment:
Don't put all behavior into a single interface if the methods are conceptually different from each other. The `draw()` method should not be in the `AreaCalculator` interface as it is conceptually different from `getArea()`.